### PR TITLE
Research notebooks for SGD sampling with learning rate scheduling, hflip augmented sampling, and SGD Init + RmsLookahead finetune in sampling.

### PR DESCRIPTION
**Pull request by jantic**
This is a rather interesting experiment where I applied learning rate scheduling to the sgd-based sampling to generate resulting images. This greatly improves upon the the baseline of just using a constant base learning rate, and it works well with 10 iterations.

### CODE REVIEW

**Comment by jantic** [comment]
Updating title to reflect the fact that I've added more research notebooks to this commit.

**Comment by jantic** [comment]
One more notebook:  SGD Init + RmsLookahead finetune in sampling.
